pages/dkp/kaptain/2.0.0/download/index.md

```diff
@@ -35,5 +33,7 @@ Kaptain install package placed in the local directory. See install instructions
 
 You will now see the `kaptain-2.0.0.tgz` file in your working directory. Follow the [Kaptain installation instructions][install] to install Kaptain. If you have problems downloading or installing Kaptain, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a>.
 
+This Docker image includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.0.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```

Comment:
This is our helm chart (not a docker image) It does contain/reference some code from MinIO however... [See slack...](https://mesosphere.slack.com/archives/C03FS0QBXRA/p1653149935535239?thread_ts=1653089333.480389&cid=C03FS0QBXRA)

Reply:
```suggestion
This Helm chart includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.0.0 are available at these URLs: [https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z](https://github.com/minio/minio/tree/RELEASE.2021-02-14T04-01-33Z) and [https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z](https://github.com/minio/minio/tree/RELEASE.2022-02-24T22-12-01Z)
```

Reply:
@lukeogg can you confirm that we need to change this in all instances? For example, also in the [tutorials PR](https://github.com/mesosphere/dcos-docs-site/pull/4486)? 
Or is it a Docker image sometimes and a Helm chart other times?

Reply:
Updating with the text from the files in the [Google docs folder](https://drive.google.com/drive/u/0/folders/1eb6tnAx6Il9YvyVKZ2Oz0Scubhuwis7P):
```suggestion
This Helm chart includes code from the MinIO Project (“MinIO”), which is © 2015-2021 MinIO, Inc. MinIO is made available subject to the terms and conditions of the [GNU Affero General Public License 3.0](https://www.gnu.org/licenses/agpl-3.0.en.html). The complete source code for the versions of MinIO packaged with Kaptain 2.0.0 are available at these URLs:
* [https://github.com/minio/operator/releases/tag/v4.0.3](https://github.com/minio/operator/releases/tag/v4.0.3)
* [https://github.com/minio/mc/releases/tag/RELEASE.2021-03-23T05-46-11Z](https://github.com/minio/mc/releases/tag/RELEASE.2021-03-23T05-46-11Z)
```